deployment/deploy.js

```diff
@@ -114,7 +114,9 @@ const deploy = async ( secretKey, network, compiler ) => {
             ),
         ]
     try {
-        await deployments[4]()
+        await deployments.reduce(
+            ( acc, f ) => acc.then( () => f() ), Promise.resolve( null )
+        )
```

Comment:
why not use `await Promise.all()` here? if they need to be sequential I like the `for (dep in deployments) { await dep() }` syntax more.